src/python/pants/backend/python/goals/coverage_py.py

```diff
@@ -160,6 +159,15 @@ def output_dir(self) -> PurePath:
     def config(self) -> Optional[str]:
         return cast(Optional[str], self.options.config)
 
+    @property
+    def config_request(self) -> ConfigFilesRequest:
+        return ConfigFilesRequest(
+            specified=self.config,
```

Comment:
Can properties like `self.config` be killed throughout the PR now to encourage single access through `ConfigFilesRequest`s?

Reply:
Not quite, we still use `self.config` to generate the argv. We could use `ConfigFiles.snapshot.files`, but in 90% of tools the config is `str | None`, whereas `Snapshot.files` is `tuple[str, ...]`, so we would need some awkwardness to assert that `snapshot.files` is len 1 even though we already know that.

Reply:
Hrm, ok. The response could also carry a function to convert to argv.